packages/preset-mini/src/_utils/utilities.ts

```diff
@@ -243,3 +243,7 @@ export function resolveVerticalBreakpoints({ theme, generator }: Readonly<Varian
 export function makeGlobalStaticRules(prefix: string, property?: string): StaticRule[] {
   return globalKeywords.map(keyword => [`${prefix}-${keyword}`, { [property ?? prefix]: keyword }])
 }
+
+export function isCSSFunction(value: string) {
+  return cssFunctions.some(fn => new RegExp(`^${fn}\\(.*\\)`).test(value))
```

Comment:
We should construct a simple regex, and cache it as a singleton. Instead of creating dynamic regex in every loop.

Reply:
wow，great insight